//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abc_O_V.h
//
// Code generation for function 'abc_O_V'
//

#ifndef ABC_O_V_H
#define ABC_O_V_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abc_O_V(double V_init, double A_init, double V_wayp, double V_max,
             double A_max, double J_max, double J_min, creal_T t[14]);

#endif
// End of code generation (abc_O_V.h)
